portal/src/graphql/portal/AuthenticationAuthenticatorSettings.tsx

```diff
@@ -277,10 +278,13 @@ const AuthenticationAuthenticatorSettings: React.FC<Props> = function Authentica
       clearEmptyObject(draftConfig);
     });
 
-    // TODO: call mutation to save config
+    const newAppConfigYaml = yaml.safeDump(newAppConfig);
+
+    await updateAppConfig(newAppConfigYaml);
```

Comment:
Add a TODO here to handle error. I think we are going to handle save error in another issue.